docs/content/reference/provider-infra/aws.md

```diff
@@ -0,0 +1,63 @@
+
+## AWS Components and Hypershift
+
+As you've seen, we've been showing all the components created, managed, not managed, etc... in the Hypershift life cycle. In this section we want to dissect who creates what and what not. It contains 4 stages:
+
+- Pre-required and Unmanaged Hosted Cluster AWS account
+- Infra managed by hypershift in Management AWS account
+- Infra managed by hypershift in Hosted Cluster AWS account
+- Infra managed by kubernetes in Hosted Cluster AWS account
+
+#### Pre-required and Unmanaged Infra in Hosted Cluster AWS account
+
+=== "Public"
+    - 1 VPC
+    - 1 DHCP Options
+    - 1 Public Subnet (for Public HostedCluster)
+    - 1 Internet Gateway
+    - 1 NAT Gateway
+    - 1 Security Group (Worker Nodes)
+    - 1 Route Tables (Public)
+    - 1 Ingress Service Load Balancer
+
+=== "Private"
+    - 1 VPC
+    - 1 DHCP Options
+    - 1 Private Subnet (for Private HostedCluster)
```

Comment:
regardless of the topology 1 Private Subnet is for data plane.
Private/Public/PublicAndPrivate affects endpoints in management side.